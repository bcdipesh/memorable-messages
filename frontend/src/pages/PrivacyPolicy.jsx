const PrivacyPolicy = () => <>PrivacyPolicy</>;

export default PrivacyPolicy;
