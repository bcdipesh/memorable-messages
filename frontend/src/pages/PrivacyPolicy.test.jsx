import React from "react";
import { BrowserRouter } from "react-router-dom";

import { render } from "@testing-library/react";

import PrivacyPolicy from "@/pages/PrivacyPolicy";

test("Renders without crashing", function () {
  render(
    <BrowserRouter>
      <PrivacyPolicy />
    </BrowserRouter>,
  );
});

test("Matches snapshot", function () {
  const { asFragment } = render(
    <BrowserRouter>
      <PrivacyPolicy />
    </BrowserRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
