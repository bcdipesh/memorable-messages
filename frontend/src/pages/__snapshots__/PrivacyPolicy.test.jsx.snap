// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Matches snapshot 1`] = `
<DocumentFragment>
  PrivacyPolicy
</DocumentFragment>
`;
